### PR TITLE
Updated name for row stream (MLDB-1754)

### PR DESCRIPTION
**Pull request by jeremybarnes**
Renamed a hard to understand field.

### CODE REVIEW

File: server/bound_queries.cc [hunk 1]
```diff
@@ -238,7 +238,7 @@ struct UnorderedExecutor: public BoundSelectQuery::Executor {
         // Do we select *?  In that case we can avoid a lot of copying
         bool selectStar = boundSelect.expr->isIdentitySelect(context);
 
-        int numRows = whereGenerator.upperBound;
+        int64_t numRows = whereGenerator.rowStreamTotalRows;
```
Comment (FinchPowers) [inline server/bound_queries.cc]: Devrait pas être unsigned?

**Comment by FinchPowers** [comment]
Commentaire mineur.
+1 conditionnel au succès des tests.

File: server/bound_queries.cc (hunk 1 above)
Comment (jeremybarnes) [inline server/bound_queries.cc]: No.  We use -1 as a guard value, which we don't want to become MAXINT if we don't check the value properly.

**Comment by jeremybarnes** [comment]
And another spurious test failure.

**Pull request merged**